website_and_docs/content/documentation/webdriver/elements/information.en.md

```diff
@@ -50,7 +54,7 @@ let result =  await driver.findElement(By.css("[name='btnK']")).isDisplayed();
 {{< /tabpane >}}
 
 {{< alert-code >}}
-for element displayedness
```

Comment:
I don't know what much difference between `element displayedness` and `isDisplayed`

Basically `element displayedness` is more generic and it is as it is according to W3C doc!

Could you please revert it!

Reply:
@harsha509 changes reverted to `element displayedness`